ghostjs-core/src/utils.js

```diff
@@ -0,0 +1,10 @@
+function getChromeFlags() {
+  let args = [];
+  if (process.env.CHROME_FLAGS) {
+    const chromeFlags = process.env.CHROME_FLAGS.split(',')
+    args = args.concat(chromeFlags)
+  }
+  return args
+}
+
+module.exports = { getChromeFlags }
```

Comment:
I am not sure why I am getting 
```
(function (exports, require, module, __filename, __dirname) { export default function getChromeFlags() {
                                                              ^^^^^^

SyntaxError: Unexpected token export
```
when I try to do `export function getChromeFlags() { ... }` which is the reason for the explicit module.exports here.